slowfs.py

```diff
@@ -15,14 +15,19 @@
 import sys
 import threading
 import time
-
-import fuse
-
+import fuse      # Filesystem in UserSpacE
+import six       # Python 2 and 3 Compatibility Library (2*3=6)
 
 def main(args):
     args = parse_args(args)
     logging.basicConfig(level=logging.DEBUG if args.debug else logging.INFO,
                         format='%(asctime)s %(levelname)s [%(name)s] %(message)s')
+    if not os.path.exists(args.mountpoint):
+        try:
+            os.mkdir(args.mountpoint)
+        except Exception as e:
+            print("Error creating mountpoint {}: {}".format(args.mountpoint, e))
+            sys.exit(1)
```

Comment:
This change is not related to supporting python 3, and I don't think it is needed. This may create the mountpoint with incorrect ownership and permissions.

Please send this change in another PR if you like.